app/editor/src/components/tabs/TabContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import { Col, Row } from 'components';
+
+import * as styled from './TabContainerStyled';
+
+export interface ITabContainerProps {
+  borderColour?: string;
```

Comment:
I generally think it's odd to have style components as separate attributes/props.  Wouldn't it be better to have standardized sizes and it all can be micromanaged through css if the dev needs to?